packages/openapi3-parser/lib/parser/oas/parseServerVariableObject.js

```diff
@@ -50,19 +50,21 @@ const parseMember = context => R.cond([
  * @returns ParseResult<Member>
  * @private
  */
-const parseServerVariableObject = (context, element) => pipeParseResult(context.namespace,
+const parseServerVariableObject = (context, element, variableName) => pipeParseResult(context.namespace,
   R.unless(isObject, createWarning(context.namespace, `'${name}' is not an object`)),
   parseObject(context, name, parseMember(context), requiredKeys, [], true),
   (object) => {
     const variable = R.or(object.get('enum'), new context.namespace.elements.String());
 
     variable.attributes.set('default', object.get('default'));
 
+    const member = new context.namespace.elements.Member(variableName, variable);
+
     if (object.hasKey('description')) {
-      variable.description = object.getValue('description');
+      member.description = object.getValue('description');
```

Comment:
```suggestion
      member.description = object.get('description');
```

I think this should actually be `get` (should've been before too though), the difference here is `getValue` is roughly equivilent to `get(...).toValue()` where we would be putting the underlying string in the member instead of the element which would contain source maps.

This should "fix" missing source map for description of server variable objects.

Reply:
Going to separate changing this into another PR

Reply:
https://github.com/apiaryio/api-elements.js/pull/530